ios/RollbarReactNative.m

```diff
@@ -19,7 +19,7 @@ @implementation RollbarReactNative
 static NSString *const REACT_NATIVE = @"react-native";
 
 + (void)initWithConfiguration:(NSDictionary*)options {
-  RollbarMutableConfig *config = ((RollbarMutableConfig *)[Rollbar configuration]);
+  RollbarMutableConfig *config = [[Rollbar configuration] mutableCopy];
   if (config) {
     updateConfiguration(config, options);
```

Comment:
I've run into the same issue following the Rollbar setup docs.  Thank you for tackling this!

If I'm following the code path correctly here, is it possible that you need to add a call to
```obj-c
[Rollbar updateWithConfiguration:config];
```
after this?  It seems like `updateConfiguration(...)` here is updating the mutable _copy_ of the configuration, but not actually applying the changes in that copy to Rollbar.